test/raft/integration/test_uv_load.c

```diff
@@ -575,14 +585,15 @@ TEST(load, openSegmentWithIncompleteBatch, setUp, tearDown, 0, NULL)
 
 /* The data directory has an open segment whose first batch is only
  * partially written. In that case the segment gets removed. */
-TEST(load, openSegmentWithIncompleteFirstBatch, setUp, tearDown, 0, NULL)
+TEST(load, openSegmentWithIncompleteFirstBatch, setUp, tearDown, 0, format_params)
 {
     struct fixture *f = data;
+
     uint8_t buf[5 * WORD_SIZE] = {
-        UV__DISK_FORMAT, 0, 0, 0, 0, 0, 0, 0, /* Format version */
+        (uint8_t)f->format_version, 0, 0, 0, 0, 0, 0, 0, /* Format version */
         0, 0, 0, 0, 0, 0, 0, 0, /* CRC32 checksums */
         0, 0, 0, 0, 0, 0, 0, 0, /* Number of entries */
-	0, 0, 0, 0, 0, 0, 0, 0, /* Local data size */
+        0, 0, 0, 0, 0, 0, 0, 0, /* Local data size */
```

Comment:
I think I am misunderstanding something, how can we have local data here where one of the `format_params` says no local data?

Reply:
Good catch---I've fixed this to use a different buf size depending on whether the test runs with format version 1 or 2. That way we get the same semantics in both cases.

Reply:
Thanks!